_field-types/supported-field-types/star-tree.md

```diff
@@ -118,17 +136,33 @@ When using the `ordered_dimesions` parameter, follow these best practices:
 
 - The order of dimensions matters. You can define the dimensions ordered from the highest cardinality to the lowest cardinality for efficient storage and query pruning. 
 - Avoid using high-cardinality fields as dimensions. High-cardinality fields adversely affect storage space, indexing throughput, and query performance.
-- Currently, fields supported by the `ordered_dimensions` parameter are all [numeric field types]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/numeric/), with the exception of `unsigned_long`. For more information, see [GitHub issue #15231](https://github.com/opensearch-project/OpenSearch/issues/15231). 
-- Support for other field types, such as `keyword` and `ip`, will be added in future versions. For more information, see [GitHub issue #16232](https://github.com/opensearch-project/OpenSearch/issues/16232).
 - A minimum of `2` and a maximum of `10` dimensions are supported per star-tree index.
 
+The `ordered_dimensions` parameter supports the following field types:
+
+  - All numeric field types excluding `unsigned_long` and `scaled_float`.
+  - `keyword` 
+  - `object`
+  - `date` which can use up to three of following calendar intervals:
+    - `year` (of era)
```

Comment:
What do we mean by "of _blank_"?

Reply:
The fields names are what defined inside "`". Extra clarification is just for information. Usually month, year etc are understood easily, but certain fields like `week` needed extra clarification like (week of week based year)